webapp/channels/src/components/login/__snapshots__/login.test.tsx.snap

```diff
@@ -28,7 +28,7 @@ exports[`components/login/Login should match snapshot with base login 1`] = `
       <h1
         className="login-body-message-title"
       >
-        Log in to your account
+        Log in to <span>your&nbsp;account</span>
```

Comment:
curious how was this generated

Reply:
The <span> is used to make sure "your account" is separated from "Log in to". "&nbsp" means non breaking space. It ensures that "your" and "account" remain in the same line.

Reply:
Yes but it should have been generated automatically via some change in test

Reply:
we arent suppose to change these manually

Reply:
Idk what that means but I've just changed the code for that specific line to avoid line break between "your" and "account".

Reply:
take a look at https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest